lib/src/firestore.dart

```diff
@@ -875,3 +879,47 @@ class DocumentQuery {
         callMethod(nativeInstance, "select", fieldPaths), firestore);
   }
 }
+
+/// A write batch, used to perform multiple writes as a single atomic unit.
+///
+/// A `WriteBatch` object can be acquired by calling `Firestore.batch()`. It
+/// provides methods for adding writes to the write batch. None of the
+/// writes will be committed (or visible locally) until `WriteBatch.commit()`
```

Comment:
nit: I'd use `[Firestore.batch]` and `[WriteBatch.commit]` style instead of backticks (references inside square brackets are clickable in IDEs and also get auto renamed with refactorings).